src/main/java/io/quarkus/bot/TriageIssue.java

```diff
@@ -86,6 +87,31 @@ void triageIssue(@Issue.Opened GHEventPayload.Issue issuePayload,
         }
     }
 
+    void notifyQE(@Issue.Labeled GHEventPayload.Issue issuePayload,
```

Comment:
Wouldn't it be more natural to have a specific `NotifyQE` class in which you would have both methods? You could share the code quite easily this way.

Reply:
ok, I wanted to follow current setup when Issue and PR are covered in separate classes, their triage methods are quite similar too. I will introduce NotifyQE class.